File: holoeasy-example-java/src/main/java/org/holoeasy/plugin/HelloWorldHologram.java
package org.holoeasy.plugin;

import org.bukkit.Location;
import org.holoeasy.HoloEasy;
import org.holoeasy.hologram.Hologram;
import org.holoeasy.line.TextLine;
import org.jetbrains.annotations.NotNull;

public class HelloWorldHologram extends Hologram {

    private final TextLine line = textLine(player -> "Hello World");

    public HelloWorldHologram(@NotNull HoloEasy lib, @NotNull Location location) {
        super(lib, location);
    }
}

File: holoeasy-example-kotlin/src/main/kotlin/org/holoeasy/plugin/HelloWorldHologram.kt
package org.holoeasy.plugin

import org.bukkit.Location
import org.holoeasy.HoloEasy
import org.holoeasy.hologram.Hologram

class HelloWorldHologram(lib : HoloEasy, location: Location) : Hologram(lib, location) {
    val line = textLine { "Hello World" }
}
